umap/static/umap/js/umap.js

```diff
@@ -786,7 +787,15 @@ U.Map = L.Map.extend({
     return U.SCHEMA[option] && U.SCHEMA[option].default
   },
 
-  getOption: function (option) {
+  getRuleOption: function (option, feature) {
```

Comment:
Can we directly use `this.rules` in the following calls?

Reply:
:+1: